libraries/common/src/main/java/androidx/media3/common/text/VoiceSpan.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright (C) 2024 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package androidx.media3.common.text;
+
+import static androidx.media3.common.util.Assertions.checkNotNull;
+
+import android.os.Bundle;
+import androidx.media3.common.util.UnstableApi;
+import androidx.media3.common.util.Util;
+import com.google.common.collect.ImmutableSet;
+import java.util.Set;
+
+/**
+ * A span representing a speaker.
+ *
+ * <p>More information on <a href="https://www.w3.org/TR/webvtt1/#webvtt-cue-voice-span">
+ *   voice spans</a>.
+ */
+@UnstableApi
+public final class VoiceSpan {
+
+  /** The speaker name. */
+  public final String speakerName;
+
+  /** The classes associated with the text. It can specify things like "first", "loud", etc. */
+  public final Set<String> classes;
```

Comment:
aiui these aren't really useful without access to the rest of the styling info. Do you have a use-case/need for these values?

Reply:
No I don't have a user for those. Do you want me to remove it?

Reply:
Yeah, let's remove it for now, if a use-case comes up later we can re-add it.

Reply:
[2a4a29b](https://github.com/androidx/media/pull/1652/commits/2a4a29b47c627cbb9d04d0845c3242456a643726)